cmd/oci-create-runtime-bundle/main.go

```diff
@@ -81,6 +82,11 @@ func newBundleCmd(stdout, stderr *log.Logger) *cobra.Command {
 It is strongly recommended to keep the default value.`,
 	)
 
+	cmd.Flags().BoolVar(
+		&v.noSameOwner, "no-same-owner", false,
+		`Extract files as yourself instead of preserving the owner and group of each entry.`,
```

Comment:
This would have to get turned around with a positive `--same-owner`.  Do cobra's boolean flags provide a way to set on/off?